server/etcdserver/server.go

```diff
@@ -655,6 +659,23 @@ func NewServer(cfg config.ServerConfig) (srv *EtcdServer, err error) {
 	return srv, nil
 }
 
+// assertNoV2StoreContent -> depending on the deprecation stage, warns or report an error
+// if the v2store contains custom content.
+func assertNoV2StoreContent(lg *zap.Logger, st v2store.Store, deprecationStage config.V2DeprecationEnum) error {
+	metaOnly, err := membership.IsMetaStoreOnly(st)
```

Comment:
no action required: since the scope of metaOnly is limited to a bool check it would be possible to reduce verbosity here.